prijateli_tree/app/routers/games.py

```diff
@@ -78,6 +78,66 @@ def get_current_round(game_id: int, db: Session = Depends(get_db)) -> int:
     return current_round
 
 
+def get_game_and_player(
+    game_id: int, player_id: int, db: Session = Depends(get_db)
+):
+    """
+    Helper function to ensure game and player exist
+    """
+    game = db.query(Game).filter_by(id=game_id).one_or_none()
+
+    if game is None:
+        raise HTTPException(
+            status_code=HTTPStatus.NOT_FOUND, detail="game not found"
+        )
+
+    player = None
+    for p in game.players:
+        if p.id == player_id:
+            player = db.query(User).filter_by(id=p.user_id).one_or_none()
+
+    if player is None:
+        raise HTTPException(
+            status_code=HTTPStatus.NOT_FOUND, detail="player not in game"
+        )
+    return game, player
+
+
+def did_player_win(
+    game_id: int,
+    player_id: int,
+    db: Session = Depends(get_db),
+    debug: bool = False,
+):
+    """
+    Helper function that determines if the player won,
+    the color of the bag and their guess
+    """
+    game, _ = get_game_and_player(game_id, player_id, db)
+    # Check if bag is red or blue
+    correct_color = get_bag_color(game.game_type.bag)
+
+    # Get the player's previous answer
+    if debug:
+        player_guess = random.choice([BALL_BLUE, BALL_RED])
```

Comment:
This should already be in place with the `app = FastAPI(debug=config.DEBUG)` line in `main.py`. Rather, you should be able to check if the app's `debug` flag is active.